src/less/components/sheet.less

```diff
@@ -97,8 +95,7 @@
             position: -webkit-sticky;
             bottom: 0;
             margin-top: auto;
-            padding-top: 2rem;
-            padding-bottom: 2rem;
+            padding: 2rem 3rem 2rem 3rem;
```

Comment:
this is 100% fine, but just wanted to mention the shortcut for it:
`padding: 2rem 3rem 2rem 3rem;` === `padding: 2rem 3rem;`
once again, it is only a shortcut, it changes absolutely nothing, it's just to save 0.5 seconds of your time next time 😅